bundles/org.openhab.binding.goveelan/README.md

```diff
@@ -0,0 +1,127 @@
+While Govee provides probably more than a hundred different lights, only the following are supported officially by the LAN API, even though others might works as well.
+
+Here is a list of the supported devices (the ones marked with * have been tested by the author)
+
+- H619Z RGBIC Pro LED Strip Lights
+- H6046 RGBIC TV Light Bars
+- H6047 RGBIC Gaming Light Bars with Smart Controller
+- H6061 Glide Hexa LED Panels
+- H6062 Glide Wall Light
+- H6065 Glide RGBIC Y Lights
+- H6066 Glide Hexa Pro LED Panel
+- H6067 Glide Triangle Light Panels
+- H6072 RGBICWW Corner Floor Lamp
+- H6076 RGBICW Smart Corner Floor Lamp (*)
+- H6073 LED Floor Lamp
+- H6078 Cylinder Floor Lamp
+- H6087 RGBIC Smart Wall Sconces
+- H6173 RGBIC Outdoor Strip Lights
+- H619A RGBIC Strip Lights With Protective Coating 5M
+- H619B RGBIC LED Strip Lights With Protective Coating
+- H619C LED Strip Lights With Protective Coating
+- H619D RGBIC PRO LED Strip Lights
+- H619E RGBIC LED Strip Lights With Protective Coating
+- H61A0 RGBIC Neon Rope Light 1M
+- H61A1 RGBIC Neon Rope Light 2M
+- H61A2 RGBIC Neon Rope Light 5M
+- H61A3 RGBIC Neon Rope Light
+- H61A5 Neon LED Strip Light 10
+- H61A8Neon Neon Rope Light 10
+- H618A RGBIC Basic LED Strip Lights 5M
+- H618C RGBIC Basic LED Strip Lights 5M
+- H6117 Dream Color LED Strip Light 10M
+- H6159 RGB Light Strip (*)
+- H615E LED Strip Lights 30M
+- H6163 Dreamcolor LED Strip Light 5M
+- H610A Glide Lively Wall Lights
+- H610B Music Wall Lights
+- H6172 Outdoor LED Strip 10m
+- H61B2 RGBIC Neon TV Backlight
+- H61E1 LED Strip Light M1
+- H7012 Warm White Outdoor String Lights
+- H7013 Warm White Outdoor String Lights
+- H7021 RGBIC Warm White Smart Outdoor String
+- H7028 Lynx Dream LED-Bulb String
+- H7041 LED Outdoor Bulb String Lights
+- H7042 LED Outdoor Bulb String Lights
+- H705A Permanent Outdoor Lights 30M
+- H705B Permanent Outdoor Lights 15M
+- H7050 Outdoor Ground Lights 11M
+- H7051 Outdoor Ground Lights 15M
+- H7055 Pathway Light
+- H7060 LED Flood Lights (2-Pack)
+- H7061 LED Flood Lights (4-Pack)
+- H7062 LED Flood Lights (6-Pack)
+- H7065 Outdoor Spot Lights
+- H6051 Aura - Smart Table Lamp
+- H6056 H6056 Flow Plus
+- H6059 RGBWW Night Light for Kids
+- H618F RGBIC LED Strip Lights
+- H618E LED Strip Lights 22m
+- H6168 TV LED Backlight
+
+## Discovery
+
+Discovery is done by scanning the devices in the Thing section. 
+
+The devices _do not_ support the LAN API support out-of-the-box.
+To be able to use the device with the LAN API, the following needs to be done (also see the "Preparations for LAN API Control" section in the [Goveee LAN API Manual](https://app-h5.govee.com/user-manual/wlan-guide)):
+
++ Start the Govee APP and add / discover the device (via Bluetooth) as described by the vendor manual
+Go to the settings page of the device 
+![govee device settings](doc/device-settings.png)
++ Note that it may take several(!) minutes until this setting comes up.
++ Switch on the LAN-Control setting.
```

Comment:
```suggestion
+ Switch on the LAN Control setting.
```

at least according to the screenshot. 🙂

Reply:
`I guess I didn't manage to convince you about the prefixes.`

You did. I changed it:

error.could-not-query-device = Could not control/query device at IP address {0}
error.offline.ip-address.missing = Handler for thing could not be added to list because ipaddress == null
error.offline.communication-error.empty-response = Offline due to receiving an empty response

`I'm sorry, I should have suggested COMMUNICATION_ERROR here.``

Like so?

<img width="816" alt="image" src="https://github.com/openhab/openhab-addons/assets/5937600/49d23641-285d-48fb-81a5-5ee8d1e8e282">

To be honest, regarding the Singleton-topic, I would leave it for the time being - I have worked several times on concurrency (even weeks on the nanoleaf binding to find out the root cause of a reoccurring issue). I definitely works and it took me hours to make it work safely. If I would go down a complete new path, this would sure put the whole binding on-hold for a number of weeks. Hence, I would ask you not to insist to change something here for the time being while I am happy to look into further after the release of the binding.

Reply:
> To be honest, regarding the Singleton-topic, I would leave it for the time being - I have worked several times on concurrency (even weeks on the nanoleaf binding to find out the root cause of a reoccurring issue). I definitely works and it took me hours to make it work safely. If I would go down a complete new path, this would sure put the whole binding on-hold for a number of weeks. Hence, I would ask you not to insist to change something here for the time being while I am happy to look into further after the release of the binding.

I'm sorry, but I will have to insist on this, since this will not only affect the binding, but can make the openHAB installation as a whole unstable. This is not easy to test manually, so even if your tests are successful, this is something that will occur randomly depending on timing etc.

Usually new bindings can take months to get reviewed, so believe me, you already have a head start here. 😉